src/functions/sortGoods.ts

```diff
@@ -0,0 +1,41 @@
+import KindsOfSort from '../constances';
+import { ISortGoods } from '../interfaces/ISortGoods';
+
+export function sortGoods({
+  sortBy,
+  setSortBy,
+  setReversed,
+  setGoods,
+  originalGoods,
+  reversed,
+}: ISortGoods): void {
+  switch (sortBy) {
+    case KindsOfSort.NO_SORT:
+      setReversed(false);
+      setGoods(originalGoods);
+      setSortBy(KindsOfSort.NO_SORT);
+      break;
+    case KindsOfSort.ALPHABET: {
```

Comment:
Add empty line between the cases
```suggestion
      break;
      
    case KindsOfSort.ALPHABET: {
```